dts/msm8916/msm8916-512mb-mtp.dts

```diff
@@ -17,5 +17,16 @@
 		/* Register EDL button as HOME button */
 		/* This is the only button on this device */
 		lk2nd,keys = <KEY_HOME 37 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+		lk2nd,match-cmdline = "* mdss_mdp.panel=1:spi:0:qcom,mdss_spi_st7735s_128128_cmd";
+	};
+
+	zhihe-uz801-v3 {
+		model = "uz801 v3.0 4G Modem Stick";
+		compatible = "zhihe,uz801-v3", "qcom,msm8916", "lk2nd,device";
+
+		/* Register the reset button as HOME button */
+		/* This is the only button on this device */
+		lk2nd,keys = <KEY_HOME 23 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_jdi_1080p_video";
```

Comment:
A panel?

Reply:
> A panel?

No, no panel. The bootloader simply passes this cmdline to kernel.